### PR TITLE
Reuse profiles/seccomp package

### PR DESCRIPTION
**Pull request by tonistiigi**
Seccomp profile loading for containerd integration was ported over from execdriver code but it meanwhile moved to a separate package.

cc @jfrazelle 

Signed-off-by: Tonis Tiigi tonistiigi@gmail.com

### CODE REVIEW

**Comment by jfrazelle** [comment]
yay thanks!

**Comment by jfrazelle** [comment]
LGTM if janky is happy

File: daemon/seccomp_linux.go [hunk 1]
```diff
@@ -3,17 +3,16 @@
 package daemon
 
 import (
-	"encoding/json"
 	"fmt"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/container"
-	"github.com/docker/engine-api/types"
+	"github.com/docker/docker/profiles/seccomp"
```
Comment (cpuguy83) [inline daemon/seccomp_linux.go]: how come we're changing this from engine-api?

Comment (tonistiigi) [inline daemon/seccomp_linux.go]: The conversion for engine-api definition to specs definition now just happens inside `profiles/seccomp` package. Loadable profile is still using the engine-api type.

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**